### PR TITLE
DOC-368: Improves scripts adds withdraw - into How To's plus extraction

### PR DESCRIPTION
**Pull request by uF4No**
- Simpler bridge scripts as they don't require the zksync-deploy package.
- Easier to run as they can be executed with ts-node.
- Adds withdraw scripts

### CODE REVIEW

**Review by Kylemclean11 — APPROVED** [review]
0789147010

**Review by Kylemclean11 — APPROVED** [review]
0789147010

**Review by Kylemclean11 — CHANGES_REQUESTED** [review]
#523

**Review by kailash-manasarovar — CHANGES_REQUESTED** [review]
-> A few clarifications needed on my side.

**Comment by dutterbutter** [comment]
Updated to fill in relevant details for withdraw script, however I the `Deposit ERC-20 tokens` script seems to hang for me on my side and could not get it to execute successfully. Can you confirm it runs successfully for you? @uF4No

File: docs/dev/how-to/withdraw-via-bridge.md [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+  }
+}
+
+main().catch((error) => {
+  console.error(`Unhandled error: ${error.message}`);
+  process.exitCode = 1;
+});
+```
+
+### 3. Run the script
```
Comment (kailash-manasarovar) [inline docs/dev/how-to/withdraw-via-bridge.md:99]:
```suggestion
### 4. Run the script
```

File: docs/dev/how-to/withdraw-via-bridge.md [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+
+```sh
+npx ts-node withdraw.ts
+```
+
+### 4. Output
```
Comment (kailash-manasarovar) [inline docs/dev/how-to/withdraw-via-bridge.md:105]:
```suggestion
### 5. Output
```

File: docs/dev/how-to/withdraw-via-bridge.md [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+  console.log(`Withdraw ERC-20 transaction sent ${withdrawErc20Handle.hash}`);
+
+}
+main().catch((error) => {
+  console.error(error);
+  process.exitCode = 1;
+});
+```
+
+### 3. Run the script
```
Comment (kailash-manasarovar) [inline docs/dev/how-to/withdraw-via-bridge.md:197]:
```suggestion
### 4. Run the script
```

File: docs/dev/how-to/withdraw-via-bridge.md [hunk 4]
```diff
@@ -0,0 +1,212 @@
...
+
+```sh
+npx ts-node withdraw-erc20.ts
+```
+
+### 4. Output
```
Comment (kailash-manasarovar) [inline docs/dev/how-to/withdraw-via-bridge.md:203]:
```suggestion
### 5. Output
```

**Review by kailash-manasarovar — DISMISSED** [review]
SGTM. Sorry, I didn't test it. If anything comes back, I'll deal with it.

File: docs/dev/how-to/deposit-via-bridge.md [hunk 5]
```diff
@@ -0,0 +1,203 @@
...
+
+## Deposit ERC-20 tokens
+
+### 1. Scaffold a new project
+
+Use the [zkSync Era cli](../../tools/zksync-cli/README.md) to set up a project.
+
+```sh
+npx zksync-cli@latest create ERC20_deposit
+```
```
Comment (uF4No) [inline docs/dev/how-to/deposit-via-bridge.md:115]: I don't think we need to use the CLI for this, we can just scaffold a project as mentioned above
```suggestion
```sh
mkdir deposit-erc20-script && cd deposit-erc20-script
npm init -y 
npm i typescript ts-node ethers@^5.7.2 zksync-web3 dotenv
```
```

File: docs/dev/how-to/withdraw-via-bridge.md [hunk 6]
```diff
@@ -0,0 +1,212 @@
...
+
+## Withdraw an ERC-20 token
+
+### 1. Set up
+
+Use the [zkSync Era cli](../../tools/zksync-cli/README.md) to set up a project.
+
+```sh
+npx zksync-cli@latest create ERC20_withdraw
+```
```
Comment (uF4No) [inline docs/dev/how-to/withdraw-via-bridge.md:124]: No need to use the CLI
```suggestion
```sh
mkdir withdraw-erc20-script && cd withdraw-erc20-script
npm init -y 
npm i typescript ts-node ethers@^5.7.2 zksync-web3 dotenv
```
```

**Comment by uF4No** [comment]
@dutterbutter yeah, deposit erc20 gets stuck although the transaction appear in the L1 explorer after a few minutes. I'll try to figure out why

**Review by Zionsammy — APPROVED** [review]

**Comment by MexicanAce** [comment]
Closing due to inactive, stale PR

**Pull request closed without merging**